tests/benchdnn/rnn/rnn.cpp

```diff
@@ -785,7 +785,7 @@ void check_known_skipped_case(const prb_t &prb, res_t *res) {
         return;
     }
 
-#if !defined(DNNL_X64)
+#if DNNL_X64 == 0
```

Comment:
Maybe play safe here and use both conditions?
```suggestion
#if !defined(DNNL_X64) || DNNL_X64 == 0
```

Reply:
Yeah, makes sense, thank you

Reply:
I think it's expected that all the arch related macros are always defined: https://github.com/oneapi-src/oneDNN/blob/57d315ebd773a6a4e43d12a6dc47894ff8d3a326/src/cpu/platform.hpp#L56-L70

So maybe it would be better to rely on that to catch issues due to undefined macros?

Reply:
@densamoilov I would put it in a standalone gtest to test it properly if we decide to test macros from platform.hpp. This is just a coincidence that it was here and we stepped on it on ARM (even our public testing on ARM didn't catch it).